package.json

```diff
@@ -668,11 +698,21 @@
                 "title": "Delete Now",
                 "category": "Kubernetes"
             },
+            {
+                "command": "extension.vsKubernetesDeleteUri",
```

Comment:
Not sure why this one is `...Uri` when Create and Apply are `...File`?

Reply:
I wanted to use vsKubernetesDeleteFile but it's already defined somewhere else :)